docs/util/config-groups.yaml

```diff
@@ -0,0 +1,50 @@
+# HTML tables will be generated for each of the config groups defined here.
+#
+# NOTE:
+#   PyYAML@6.0.1 will silently overwrite the value of a key if it is defined
+#   multiple times. Be sure not to accidentally define a group more than once.
+#   See: https://github.com/yaml/pyyaml/issues/165
+sql-tuning-caching-data:
+  - spark.sql.inMemoryColumnarStorage.compressed
+  - spark.sql.inMemoryColumnarStorage.batchSize
+sql-tuning-broadcast:
+  - spark.sql.autoBroadcastJoinThreshold
+  - spark.sql.broadcastTimeout
```

Comment:
I think some config are intersections in the division of ownership.
Such as: `spark.sql.adaptive.autoBroadcastJoinThreshold`.
It seems we can't division them indecisively.

Reply:
Looks good.